prusa_slicer_post_processing_script.py

```diff
@@ -65,7 +65,7 @@ def makeFullSettingDict(gCodeSettingDict:dict) -> dict:
         "aboveArcsPerimeterPrintSpeed":3*60, #Unit: mm/min
         "applyAboveFanSpeedToWholeLayer":True,
         "CoolingSettingDetectionDistance":5, #if the gcode line is closer than this distance to an infill polygon the cooling settings will be applied. Unit:mm
-        "specialCoolingZdist":3, #use the special cooling XX mm above the arcs.
+        "specialCoolingZdist":3, #use the special cooling XX mm above the arcs. Set to a negative value to disable (not recommended).
```

Comment:
Tells the user how to disable Hilbert curves. Could possibly stand to be more specific...